test/code/plugins/out_oms_changetracking_file_systest.rb

```diff
@@ -0,0 +1,179 @@
+require 'fluent/test'
+require_relative ENV['BASE_DIR'] + '/source/ext/fluentd/test/helper'
+require_relative ENV['BASE_DIR'] + '/source/code/plugins/out_oms_changetracking_file'
+require_relative 'omstestlib'
+require_relative 'out_oms_systestbase'
+
+class TestableOutChangeTrackingFile < Fluent::OutChangeTrackingFile
```

Comment:
Instead, I created a testable class, which has mock of append_blob method. so I test the parameters of the method and make sure they as expected. This is the best I can do at this point